### PR TITLE
[SPARK-LLAP-133][BRANCH-1.6] Access control for DESC TABLE

### PR DESCRIPTION
**Pull request by merlintang**
## What changes were proposed in this pull request?

add the access control for des table 

## How was this patch tested?

under testing

### CODE REVIEW

File: src/main/scala/org/apache/spark/sql/hive/llap/LlapContext.scala [hunk 1]
```diff
@@ -124,6 +122,28 @@ class LlapContext(sc: SparkContext,
...
+  override protected[sql] def executePlan(plan: LogicalPlan): super.QueryExecution = {
+    new QueryExecution(plan)
+    super.executePlan(plan)
+  }
+
+  /** Extends QueryExecution with hive specific features. */
+  private[sql] class QueryExecution(logicalPlan: LogicalPlan)
+    extends org.apache.spark.sql.execution.QueryExecution(this, logicalPlan) {
+    /**
+      * Access control for the desc table
```
Comment (dongjoon-hyun) [inline src/main/scala/org/apache/spark/sql/hive/llap/LlapContext.scala]: indentation?

Comment (merlintang) [inline src/main/scala/org/apache/spark/sql/hive/llap/LlapContext.scala]: thanks Dongjoon, let me fix this.

**Comment by merlintang** [comment]
@dongjoon-hyun  if no further comments, I will merge this patch.

**Comment by dongjoon-hyun** [comment]
Where is the test result?

**Comment by merlintang** [comment]
I have tested on a setup cluster, but the cluster was gone. let me create a other cluster and post the result here.

**Comment by merlintang** [comment]
I close this one, since I create a other PR with test result.

**Comment by merlintang** [comment]
close this one since i have opened other PR.

**Pull request closed without merging**